src/SockPerf.cpp

```diff
@@ -3113,7 +3127,7 @@ int bringup(const int *p_daemonize)
 		else
 			log_msg("VMA Extra API found - using VMA's receive zero copy and messages filter APIs");
 
-		_vma_pkts_desc_size = sizeof(struct vma_packets_t) + sizeof(struct vma_packet_t) + sizeof(struct iovec) * 16;
+		_vma_pkts_desc_size = sizeof(struct vma_completion_t) * 4;
 #else
```

Comment:
@AlaAyesh sockperf has zcopy support that is based recv callback. Probably you mixed this one with vma_poll()